docs/istio-custom-ca-with-csr.md

```diff
@@ -6,7 +6,7 @@ certificates using an Issuer provided by the Trusted Certificate Service (TCS).
 
 ## Prerequisites for running this example:
 
-  - istioctl
+  - istioctl (version 1.13 or later)
```

Comment:
Does this mean we do not support Istio < 1.13?

Reply:
I think we should mention what is the minimum Istio version we support if applicable.

Reply:
I have tried to test this istio 1.12 but something is failing. Need to dig in more what is wrong. But at least the same setup (istio config file) does not work with with istio 1.12.